src/data/mv_polynomial/basic.lean

```diff
@@ -0,0 +1,862 @@
+/-
+Copyright (c) 2017 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Johan Commelin, Mario Carneiro, Shing Tak Lam
+-/
+import data.polynomial.eval
+import data.equiv.ring
+import data.equiv.fin
+import data.set.disjointed
+import tactic.omega
```

Comment:
```suggestion
```
I might have missed it, but is `omega` used anywhere here?

Reply:
Oh, good point, I didn't move the imports from `basic` to the new files.